uiServices/src/testCaseServices.go

```diff
@@ -128,12 +129,17 @@ func putTestCase(rw http.ResponseWriter, req *http.Request) {
 func getAllTestCases(rw http.ResponseWriter, req *http.Request) {
 
 	testCasePathDir := getTestCaseHeader(req)
-	if len(testCasePathDir) <= 1 {
-		logrus.Error("No file directory entered")
-		rw.WriteHeader(http.StatusBadRequest)
+
+	if !IsPathDirValid(testCasePathDir, rw) {
 		return
 	}
 
+	// if len(testCasePathDir) <= 1 {
```

Comment:
remove commented-out code

Reply:
Leave a commen@jenny-flynn  here is the func ValidateFileNameAndHeader, based on your feedback

**_In configServices.go the function is split in two:_**
```
func IsHeaderValid(header string) error {
	if len(header) <= 1 {
		logrus.Error("No Header Path Found")
		return fmt.Errorf("No Header Path Found")
	}
	return nil
}

func IsNameValid(name string) error {
	if len(name) < 1 {
		logrus.Error("File Name is Empty")
		return fmt.Errorf("File Name is Empty")
	}
	return nil
}

```
**and in the main function**:

```
if err := IsHeaderValid(testSuitePathDir); err != nil {
		rw.WriteHeader(http.StatusBadRequest)
		return
	}

	if  err:= IsNameValid(testSuiteName); err != nil {
		rw.WriteHeader(http.StatusBadRequest)
		return
	}
```